app/assets/stylesheets/providers/search_results.scss

```diff
@@ -4,15 +4,23 @@
   display: none;
 }
 
-.proceeding-item,
-.organisation-item {
+.proceeding-item {
   display: block;
 
   .control-column {
     text-align: right;
   }
 }
 
+.organisation-item {
+  mark {
+    margin-right: 1px;
+    padding: 2px 0 0;
+    border-bottom: 3px solid #ffdd00;
+    background-color: #fff2d3;
```

Comment:
Nice.

Could we use the govuk color palette instead?

`govuk-colour("yellow")` instead of `#ffdd00`

The background color isnt on the govuk palette, but you could choose something close to it: https://design-system.service.gov.uk/styles/colour/